python/ray/_private/worker.py

```diff
@@ -688,7 +688,7 @@ def set_mode(self, mode):
     def set_load_code_from_local(self, load_code_from_local):
         self._load_code_from_local = load_code_from_local
 
-    def put_object(self, value, object_ref=None, owner_address=None):
+    def put_object(self, value, object_ref=None, owner_address=None, is_mutable=False):
```

Comment:
```suggestion
    def put_object(self, value, object_ref=None, owner_address=None, _is_mutable=False):
```

Also should document this as experimental / for accelerated dags.